pyscf/pbc/adc/test/test_k_vs_supercell_ip.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2014-2019 The PySCF Developers. All Rights Reserved.
```

Comment:
Add more tests for C, add tests for degeneracy of roots

Reply:
Added